infra/core/app-configuration/config-data/config-data.json

```diff
@@ -0,0 +1,86 @@
+[
+  {
+    "key": "ENVIRONMENT",
+    "value": "#{{ lower(environment) }}",
+    "label": "Platform",
+    "contentType": "text/plain"
+  },
+  {
+    "key": "APPLICATIONS_ACR_NAME",
+    "value": "#{{ infraResourceNamePrefix }}#{{ nc_resource_containerregistry }}#{{ nc_instance_regionid }}01",
+    "label": "Platform",
+    "contentType": "text/plain"
+  },
+  {
+    "key": "FLUX_CONFIG_NAMESPACE",
+    "value": "flux-config",
+    "label": "Platform",
+    "contentType": "text/plain"
+  },
+  {
+    "key": "SERVICEBUS_RG",
+    "value": "#{{ servicesResourceGroup }}",
+    "label": "Platform",
+    "contentType": "text/plain"
+  },
+  {
+    "key": "SERVICEBUS_NS",
```

Comment:
would maybe add in SHRD_ at the start to follow convention ?

Reply:
and make NS == NAMESPACE maybe ?

Reply:
Done